docs/docs/configuration/alpha_config.md

```diff
@@ -204,6 +221,28 @@ ClaimSource allows loading a header value from a claim within the session
 | `prefix` | _string_ | Prefix is an optional prefix that will be prepended to the value of the<br/>claim if it is non-empty. |
 | `basicAuthPassword` | _[SecretSource](#secretsource)_ | BasicAuthPassword converts this claim into a basic auth header.<br/>Note the value of claim will become the basic auth username and the<br/>basicAuthPassword will be used as the password value. |
 
+### Cookie
+
+(**Appears on:** [AlphaOptions](#alphaoptions))
+
+Cookie contains configuration options relating session and CSRF cookies
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| `name` | _string_ | Name is the name of the cookie |
+| `secret` | _string_ | Secret is the secret used to encrypt/sign the cookie value |
```

Comment:
Secret should be a secret source right? So it can be literal, from env or from file?

Which then means we don't need secret file?